pages/c/_cluster/apps/charts/index.vue

```diff
@@ -17,10 +19,50 @@ import { removeObject, addObject, findBy } from '@/utils/array';
 import { compatibleVersionsFor, filterAndArrangeCharts } from '@/store/catalog';
 import { CATALOG } from '@/config/labels-annotations';
 
+const sliderData = [
```

Comment:
Please replace this fictitious data with actual charts as defined in the requirement:

> For now, create a method 'getFeaturedCharts' which will should return at mot 5 charts to show in the featured carousel. For development, you can hard code this function to select any 5 charts from those installed - once the chart annotations are in place, we an update the function to use real data to get the actual featured charts to display. The function should arrange the 5 charts, so that the most recently added/updated chart is shown in the middle
>
> Charts in the carousel should be shown in order 5 3 1 2 4 from a list of featured charts which is ordered from most recently added/updated to least - i.e. the newest chart of interest should be in the middle (1)